src/main/java/me/zeroeightsix/kami/module/modules/misc/AutoObsidian.kt

```diff
@@ -64,6 +59,10 @@ class AutoObsidian : Module() {
     private var tickCount = 0
     private var openTime = 0L
 
+    override fun isActive(): Boolean {
```

Comment:
what is this overriding?

Reply:
see https://github.com/kami-blue/client/blob/d69e0a806e9e4d2539556411510421cf17299c1b/src/main/java/me/zeroeightsix/kami/module/Module.kt#L143-L145

Reply:
Ahh makes sense.